db/migrate/20170604144747_add_foreign_keys_for_accounts.rb

```diff
@@ -0,0 +1,36 @@
+class AddForeignKeysForAccounts < ActiveRecord::Migration[5.1]
+  def change
+    add_foreign_key :statuses, :accounts, on_delete: :cascade
```

Comment:
There is still a missing foreign key for `in_reply_to_account_id`. Maybe the action for this one should be `nullify`.